test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -70,6 +74,8 @@
 
     private static final Random RND = RandomFactory.getRandom();
 
+    private static Collection<Path> tempFiles;
+
```

Comment:
Is there a reason you chose a LinkedList over a List here?  As mentioned below, you could initialize  tempFiles here.

Please change Collection -> LinkedList or if you use a List to ArrayList()

Reply:
If you change from an interface to a particular implementation you give the reader the impression that the code will only work with this particular implementation, and you give the impression that the code will only work in a particular sequence when using List or LinkedList, which both is not true. The Collection interface was chosen deliberately to clearly tell the reader that both is completely irrelevant and may be changed at any time if needed. If I do what you want, others will fear to break something. So i would like to stay with the code as-is.

Reply:
What *actual* benefit will you reach in this *particular* source code by using array list instead of linked list?

Reply:
There is really no need to use a LinkedList so the use of a List is just simpler.

Reply:
There also is no need to use an ArrayList, so how does you answer explain your request?